packages/react-ui/src/components/HatchifyDisplays/HatchifyDisplays.tsx

```diff
@@ -28,6 +28,17 @@ export const HatchifyExtraDisplay: React.FC<HatchifyExtraDisplayProps> = () => {
   return null
 }
 
+export type HatchifyEmptyListProps = {
+  label: string
+  after?: string
+} & ({ render: () => JSX.Element } | { ValueComponent: ValueComponent })
```

Comment:
- we can remove `label` and `after`
- `render` leave as is
- `ValueComponent`should be `EmptyComponent: EmptyComponent` because the `ValueComponent` _type_ expects certain props for displaying attributes which we don't need here